src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/IntegrationTestSupport.java

```diff
@@ -19,4 +36,52 @@ static void setSourcePath(String path, String propertyPlaceHolder) throws Except
             System.setProperty(propertyPlaceHolder, "file:" + loadFile.replaceFirst("/", ""));
         }
     }
+
+    static List<JudicialOfficeAuthorisation> getFileAuthorisationObjectsFromCsv(String inputFilePath)  {
+        List<JudicialOfficeAuthorisation> authorisations = new LinkedList<>();
+        try {
+            File file = ResourceUtils.getFile(inputFilePath);
+            InputStream inputStream = new FileInputStream(file);
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+            authorisations = bufferedReader.lines().skip(1).map(line -> mapJudicialOfficeAuthorisation(line)).collect(Collectors.toList());
+            bufferedReader.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return authorisations;
+    }
+
+    static JudicialOfficeAuthorisation mapJudicialOfficeAuthorisation(String line) {
+        List<String> columns = Arrays.asList(line.split("\\,", -1));
+        JudicialOfficeAuthorisation judicialOfficeAuthorisation = new JudicialOfficeAuthorisation();
+        judicialOfficeAuthorisation.setElinksId(handleNull(columns.get(0), false));
+        judicialOfficeAuthorisation.setJurisdiction(handleNull(columns.get(1), false));
+        judicialOfficeAuthorisation.setTicketId(isBlank(columns.get(2)) ? null : Long.parseLong(columns.get(2)));
+        judicialOfficeAuthorisation.setStartDate(handleNull(columns.get(3), true));
+        judicialOfficeAuthorisation.setEndDate(handleNull(columns.get(4), true));
+        judicialOfficeAuthorisation.setCreatedDate(handleNull(columns.get(5), true));
+        judicialOfficeAuthorisation.setLastUpdated(handleNull(columns.get(6), true));
+        judicialOfficeAuthorisation.setLowerLevel(handleNull(columns.get(7), false));
+        return judicialOfficeAuthorisation;
+    }
+
+    static String handleNull(String fieldValue, boolean timeStampField) {
+        if (isBlank(fieldValue) && !timeStampField) {
+            return BLANK;
+        } else if (timeStampField && isBlank(fieldValue)) {
```

Comment:
i did not get why we need this method?

Reply:
required to avoid ternary operator. I wanted blank string when null and also same for timestamp field